cumulus/parachains/integration-tests/emulated/tests/assets/asset-hub-westend/src/tests/reserve_transfer.rs

```diff
@@ -1132,8 +1145,666 @@ fn reserve_transfer_native_asset_from_para_to_para_through_relay() {
 		<ForeignAssets as Inspect<_>>::balance(relay_native_asset_location, &receiver)
 	});
 
-	// Sender's balance is reduced by amount sent plus delivery fees
+	// Sender's balance is reduced by amount sent plus delivery fees.
 	assert!(sender_assets_after < sender_assets_before - amount_to_send);
+	// Receiver's balance is increased by `amount_to_send` minus delivery fees.
+	assert!(receiver_assets_after > receiver_assets_before);
+	assert!(receiver_assets_after < receiver_assets_before + amount_to_send);
+}
+
+// =========================================================================
+// ======= Reserve Transfers - Sufficient Asset - AssetHub->Parachain ======
+// =========================================================================
+/// Reserve Transfers of a sufficient asset from System Parachain to Parachain should
+/// work without using any other token.
+#[test]
+fn reserve_transfer_sufficient_assets_from_system_para_to_para() {
```

Comment:
This is already kind of covered by `reserve_transfer_sufficient_asset_from_para_to_para_through_asset_hub` - not sure if we need to maintain it

Reply:
I originally made this test to make sure you could pay delivery fees locally with another asset, but now that's not the case. I left the test but put a TODO to make sure this is addressed in the future but I can just remove the tests.